### PR TITLE
Edit API key

### PR DESCRIPTION
**Pull request by AramAlsabti**

### CODE REVIEW

File: src/app/admin/admin.module.ts [hunk 1]
```diff
@@ -90,7 +90,6 @@ import { ApiKeyEditComponent } from './api-key/api-key-edit/api-key-edit.compone
     ApiKeyComponent,
     ApiKeyListComponent,
     ApiKeyTableComponent,
-    ApiKeyEditComponent,
```
Comment (augusthjerrild) [inline src/app/admin/admin.module.ts:93]: Is there a reason for this being deleted? I notice that the Organisation, Permission and User also are exporting the edit component.

File: src/app/admin/api-key/api-key-edit/api-key-edit.component.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ export class ApiKeyEditComponent implements OnInit {
   public formFailedSubmit = false;
   public permissions: PermissionResponse[] = [];
   private organizationId: number;
+  id: number;
```
Comment (augusthjerrild) [inline src/app/admin/api-key/api-key-edit/api-key-edit.component.ts:33]: This should be either private or public, compared to the rest of the properties :-)

File: src/app/admin/api-key/api-key-edit/api-key-edit.component.ts [hunk 3]
```diff
@@ -79,8 +83,17 @@ export class ApiKeyEditComponent implements OnInit {
       );
   }
 
+  private getApiKey(id: number) {
+    this.apiKeyService.get(id).subscribe((key) => {
+      this.apiKeyRequest = new ApiKeyRequest();
```
Comment (augusthjerrild) [inline src/app/admin/api-key/api-key-edit/api-key-edit.component.ts:88]: In line 20, apiKeyRequest is already created with new ApiKeyRequest(). Is there a reason that this is done again in this place? Furthermore, apiKeyRequest in line 20 could be declared at public compared to the other properties.

File: src/app/admin/api-key/api-key.model.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { PermissionResponse } from '../permission/permission.model';
 export class ApiKeyRequest {
   id: number;
   name: string;
-  permissions?: PermissionResponse[];
+  permissions?: number[];
```
Comment (augusthjerrild) [inline src/app/admin/api-key/api-key.model.ts:6]: Maybe "permissions" should be changed to "permissionIds"

**Review by augusthjerrild — CHANGES_REQUESTED** [review]
Overall looking good. A few minor changes/considerations should be made.

File: src/app/admin/admin.module.ts (hunk 1 above)
Comment (AramAlsabti) [reply src/app/admin/admin.module.ts:93]: It's strictly not necessary. It's not being imported anywhere else, but I'll add it back for consistency

**Review by augusthjerrild — APPROVED** [review]
Looking good. Good job :-)

**Pull request merged**